From d61222f24d024fd9ee5063d81f3ce37cd9a9bf8b Mon Sep 17 00:00:00 2001
From: Lionel Debieve <lionel.debieve@foss.st.com>
Date: Tue, 2 Nov 2021 11:11:41 +0100
Subject: [PATCH 0099/1695] mk: config: allow to use lightweight tags instead
 of annotated ones

Do not use the last annotated tags, use the last available
one instead also if it's a lightweight one.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 84a59249d84b79e5ec794d0832a334cfad7bf629 mk: config: allow to use lightweight tags instead of annotated ones]
Change-Id: I1d86bfc774cfca7d7e58fef2470ac226663c2c8b
Signed-off-by: Lionel Debieve <lionel.debieve@foss.st.com>
[etienne: picked from rebase-3.19 development branch]
Signed-off-by: Etienne Carriere <etienne.carriere@foss.st.com>
---
 mk/config.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mk/config.mk b/mk/config.mk
index 7b6eb8efd..fa32407e9 100644
--- a/mk/config.mk
+++ b/mk/config.mk
@@ -125,7 +125,7 @@ CFG_OPTEE_REVISION_MINOR ?= 0
 CFG_OPTEE_REVISION_EXTRA ?=
 
 # Trusted OS implementation version
-TEE_IMPL_VERSION ?= $(shell git describe --always --dirty=-dev 2>/dev/null || \
+TEE_IMPL_VERSION ?= $(shell git describe --always --tags --dirty=-dev 2>/dev/null || \
 		      echo Unknown_$(CFG_OPTEE_REVISION_MAJOR).$(CFG_OPTEE_REVISION_MINOR))$(CFG_OPTEE_REVISION_EXTRA)
 ifeq ($(CFG_OS_REV_REPORTS_GIT_SHA1),y)
 TEE_IMPL_GIT_SHA1 := 0x$(shell git rev-parse --short=8 HEAD 2>/dev/null || echo 0)
-- 
2.47.2

